packages/aws-cdk/test/api/hotswap/lambda-versions-aliases-hotswap-deployments.test.ts

```diff
@@ -8,14 +8,26 @@ let hotswapMockSdkProvider: setup.HotswapMockSdkProvider;
 
 beforeEach(() => {
   hotswapMockSdkProvider = setup.setupHotswapTests();
-  mockUpdateLambdaCode = jest.fn();
+  mockUpdateLambdaCode = jest.fn().mockReturnValue({});
   mockPublishVersion = jest.fn();
   mockUpdateAlias = jest.fn();
   hotswapMockSdkProvider.stubLambda({
     updateFunctionCode: mockUpdateLambdaCode,
     publishVersion: mockPublishVersion,
     updateAlias: mockUpdateAlias,
     waitFor: jest.fn(),
+  }, {
+    // these are needed for the waiter API that the Lambda service hotswap uses
+    api: {
+      waiters: {},
+    },
+    makeRequest() {
+      return {
+        promise: () => Promise.resolve({}),
+        response: {},
+        addListeners: () => {},
+      };
+    },
```

Comment:
We keep repeating this block in a lot of the tests. What do you think of actually updating the `stubLambda()` method in `HotswapMockSdkProvider` to do it for us?